clap_complete/tests/testsuite/dynamic.rs

```diff
@@ -19,20 +19,48 @@ macro_rules! complete {
 #[test]
 fn suggest_subcommand_subset() {
     let mut cmd = Command::new("exhaustive")
-        .subcommand(Command::new("hello-world"))
-        .subcommand(Command::new("hello-moon"))
-        .subcommand(Command::new("goodbye-world"));
+        .subcommand(Command::new("hello-world").visible_alias("hello-world-foo").alias("hidden-world").visible_long_flag_alias("--world").long_flag_alias("--hidden-world").visible_short_flag_alias('w').short_flag_alias('W'))
```

Comment:
Instead of reusing an existing test, we should make sure this functionality is on `aliases_command` and create a new test that covers that.